experiment/secret-sync-controller/client_util/client_util.go

```diff
@@ -0,0 +1,46 @@
+package client_util
```

Comment:
avoid snake case, consider instead making this something like `secret-sync-controller/client/client.go`

Reply:
got rid of all snake case filename/package name/variables.
Is there any circumstance that we would use snake case?